### ISSUE TITLE
Nipple tenderness

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10090]
**Preferred term label:**
Nipple tenderness

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0240521

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mammary duct ectasia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0020226
